godot-core/src/builtin/glam_helpers.rs

```diff
@@ -9,12 +9,14 @@
 //
 // Nice:
 //   self.glam2(&with, |a, b| a.dot(b))
-//   self.glam2(&with, glam::f32::Quat::dot)
+//   self.glam2(&with, glam::Real::Quat::dot)
```

Comment:
search&replace mistake?

Reply:
oh yeah, i spotted a couple earlier, missed this one